guides/common/modules/proc_creating-a-content-filter-for-yum-content.adoc

```diff
@@ -1,8 +1,8 @@
-[id="Creating_a_Content_Filter_{context}"]
-= Creating a Content Filter
+[id="Creating_a_Content_Filter_for_YUM_Content_{context}"]
+= Creating a Content Filter for YUM Content
```

Comment:
This might raise some eyebrows for Satellite folks. Can we leave the ID but do this... 
```suggestion
ifdef::satellite[]
= Creating a Content Filter
endif::[]
ifndef::satellite[]
= Creating a Content Filter for YUM Content
endif::[]
```

Reply:
You can also have content filters for container images as far as I can see.

Reply:
Ahh, if that's the case, you can ignore this, if you just s/YUM/Yum :)

Reply:
That has been fixed, yes.